packages/bpk-component-fieldset/src/BpkFieldset-test.js

```diff
@@ -198,16 +199,14 @@ describe('BpkFieldset', () => {
   });
 
   it('should reject label property when label is omitted', () => {
-    expect(
-      BpkFieldset.propTypes.label({}, 'label', 'BpkFieldset').toString(),
-    ).toEqual('Error: `label` is required when `isCheckbox` is false.'); // eslint-disable-line max-len
+    expect(BpkFieldset.propTypes.label({}, 'label').toString()).toEqual(
```

Comment:
Were these tests previously broken?

Reply:
No but the extra argument shouldn't have been there